app/lib/choicesOverview.js

```diff
@@ -0,0 +1,14 @@
+function choicesOverviewUrl(inputList) {
```

Comment:
Given the module name is `choicesOverview` I'd suggest changing the function's name to `addUrl` or something similar to describes what it is doing.

This function needs some unit tests.

Reply:
@st3v3nhunt wrote:
> Given the module name is choicesOverview I'd suggest changing the function's name to addUrl or something similar to describes what it is doing.
Done